osmtm/templates/project.edit.mako

```diff
@@ -36,6 +36,43 @@
   </div>
   </div>
 </div>
+
+<!-- Modal -->
+<div class="modal fade" id="invalidateAllModal" tabindex="-1" role="dialog" aria-labelledby="invalidateAll">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title" id="myModalLabel">${_('Invalidate All')}</h4>
+      </div>
+      <div class="modal-body">
+        <p>
+          ${_("This will mark all tasks currently marked as 'done' as invalid. Please use this only if you are sure of what you are doing.")}
+        </p>
+        <div class="text-center">
+          <div class="form-group">
+            <textarea id="project_invalidate_comment" name="invalidate_all_comment" class="form-control" placeholder="${_('Leave a comment')}" rows="2"></textarea>
+
+          </div>
+          <div class="form-group">
+            Please enter the number ${project.id} in the box:
+            <input id="project_invalidate_challenge_id" />
+          </div>
+          <p class="errors">
+
+          </p>
+          <a class="btn btn-danger btn-lg btn-invalidate-all">
+             <span class="glyphicon glyphicon-share-alt"></span>&nbsp;
+             ${_('Invalidate all done tasks')}
+          </a>
+        </div>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
```

Comment:
Makes sense. Done here: https://github.com/osm-in/osm-tasking-manager2/commit/323396c467d3603377acc12354f8a72a29439aa4